test/Microsoft.Framework.Runtime.Tests/TestSample/project.lock.sample

```diff
@@ -0,0 +1,2180 @@
+{
```

Comment:
Can we generate this dynamically by calling `dnu restore` in tests? Then we don't need to update this later when we change lock file format.

You can generate a lock file for `HelloWorld` sample project by calling this: https://github.com/aspnet/dnx/blob/dev/test/Bootstrapper.FunctionalTests/BootstrapperTestUtils.cs#L35